keyboards/system76/launch_1/config.h

```diff
@@ -18,96 +18,91 @@
 #pragma once
 
 #include "config_common.h"
+#ifndef __ASSEMBLER__
```

Comment:
this shouldn't ben necessary

Reply:
Unfortunately, it is necessary. Unfamiliar with the build dependencies well enough to implement a better workaround. Tried with AVR LIBC 5.4.0 and 7.3.0. Build error output:
```log
Assembling: platforms/avr/xprintf.S                                                                 [ERRORS]
 | 
 | /usr/lib/avr/include/stdint.h: Assembler messages:
 | /usr/lib/avr/include/stdint.h:125: Error: unknown opcode `typedef'
...
 | /usr/lib/avr/include/stdint.h:282: Error: unknown opcode `typedef'
 | quantum/eeconfig.h:92: Error: unknown opcode `_bool'
 | quantum/eeconfig.h:95: Error: unknown opcode `void'
 | quantum/eeconfig.h:104: Error: unknown opcode `uint8_t'
...
 | quantum/eeconfig.h:134: Error: unknown opcode `void'
 | 
gmake[1]: *** [builddefs/common_rules.mk:359: .build/obj_system76_launch_1_default/platforms/avr/xprintf.o] Error 1
```

Reply:
5.4 is old enough, that it doesn't support a lot.  It shouldn't be used, period. 

We specifically recommend 8.x for a reason.  And a big part of that is supported feature set of the compilers.  

So, yeah, this shouldn't be necessary, at all.

Reply:
Same behavior with 7.3.0 as with 8.3.0, 9.2.0, 10.1.0, 11.1.0 and 12.1.0. It does not appear to be a compiler feature issue. As mentioned, there is something abstracted into the build system that is manifesting in these side effects. Consequently, hardcoded the EEPROM size variable to work around the need to include any files in `config.h`.